### ISSUE TITLE
Soundgrain 6.0.1 wxAssertionError, will not start

### ISSUE THREAD

**Issue by oatmealraisin** [belangeo/soundgrain#24]
```
pyo version 0.9.0 (uses single precision)
Traceback (most recent call last):
  File "SoundGrain.py", line 59, in <module>
    app = SoundGrainApp(redirect=False)
  File "SoundGrain.py", line 41, in __init__
    screen_size=(sysx, sysy))
  File "/home/ryan/documents/sources/SoundGrain_6.0.1-src/Resources/MainFrame.py", line 220, in __init__
    self.controls = ControlPanel(self, self.panel)
  File "/home/ryan/documents/sources/SoundGrain_6.0.1-src/Resources/ControlPanel.py", line 192, in __init__
    rec1Box.Add(self.but_folder, 1, wx.ALIGN_CENTER_VERTICAL | wx.EXPAND | wx.RIGHT, 10)
wx._core.wxAssertionError: C++ assertion "!(flags & (wxALIGN_BOTTOM | wxALIGN_CENTRE_VERTICAL))" failed at /tmp/pip-install-ygicj3e5/wxpython/ext/wxWidgets/src/common/sizer.cpp(2111) in DoInsert(): Vertical alignment flags are ignored with wxEXPAND
```

```
Markdown==3.3.2
  - importlib-metadata [required: Any, installed: 2.0.0]
    - zipp [required: >=0.5, installed: 3.3.1]
pyo==0.9.0
wxPython==4.1.0
  - numpy [required: Any, installed: 1.19.2]
  - pillow [required: Any, installed: 8.0.0]
  - six [required: Any, installed: 1.15.0]
```

Same happens with wxPython 4.0.1

`Linux lungo 5.9.1-arch1-1 #1 SMP PREEMPT Sat, 17 Oct 2020 13:30:37 +0000 x86_64 GNU/Linux`

**Comment by belangeo** [comment]
Pyo 0.9.0 is 2 years and a half old ! You should really update to 1.0.3 (now hosted on pypi.org). Also, if you want to run Soundgrain from sources, you'll need wxpython 4.1.0 (there was some breaking changes).

**Comment by oatmealraisin** [comment]
You may want to cut a new release, because the 6.0.1 doesn't work with the
new pyo import stuff.

I'm nearly certain I reproduce the same results with pyo 1.03 and wx 4.1.0,
but I'll try again and get back.

Thanks for the quick response!

**Comment by belangeo** [comment]
Oh, my bad, I thought I made the fixes for wxpython 4.1.0 but no, not for soundgrain yet. I'll do it soon. Meanwhile, you can install wxpython 4.0.7 (it should not happen with version prior to 4.1.0).

**Comment by oatmealraisin** [comment]
Finally got it to work with wxpython 4.0.7 from my package manager. Couldn't get the `pip install wxPython==4.0.7` to work at all though, lol. Wheel wouldn't build.
